.github/workflows/automated-prs-manager.yaml

```diff
@@ -68,26 +68,27 @@ jobs:
           fi
 
           # If all checks passed, approve and merge.
-
-          if gh pr checks "${{ matrix.pr.url }}" ; then
-            echo "All tests passed. Approving and merging."
-            echo -e "LGTM :thumbsup: \n\nThis PR was automatically approved and merged by the [automated-prs-manager](https://github.com/replicatedhq/troubleshoot/blob/main/.github/workflows/automated-prs-manager.yaml) GitHub action" > body.txt
-            gh pr review --approve "${{ matrix.pr.url }}" --body-file body.txt
-            gh pr merge --auto --squash "${{ matrix.pr.url }}"
-            sleep 10
-            exit 0
-          else
-            echo "Some checks did not pass. Skipping."
-            exit 0
+          if gh run view "$run_id" --json jobs -q '.jobs[] | select(.name == "validate-success") | .conclusion' | grep -q "success"; then
+            if gh pr checks "${{ matrix.pr.url }}" ; then
```

Comment:
if gh pr checks "${{ matrix.pr.url }}" --required; then

maybe add --required to have less check, and only dependents on the GitHub PR Checks are labeled with required

Reply:
I don't want to auto merge anything that fails a test - it'll mean all future PRs fail the same test